sync/linux/controlplane.sh

```diff
@@ -129,12 +127,24 @@ sudo docker tag k8s.gcr.io/etcd:3.4.13-0 gcr.io/k8s-staging-ci-images/etcd:3.4.1
 sudo docker tag k8s.gcr.io/pause:3.4.1 gcr.io/k8s-staging-ci-images/pause:3.4.1
 sudo docker tag k8s.gcr.io/coredns/coredns:v1.8.0 gcr.io/k8s-staging-ci-images/coredns/coredns:v1.8.0
 
+cat << EOF > /var/sync/shared/kubeadm.yaml
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: InitConfiguration
+localAPIEndpoint:
+  advertiseAddress: 10.20.30.10
```

Comment:
maybe we should pass this from Vagrantfile via args?

Reply:
I agree. Data flow should be consistent.